packages/lesswrong/components/analytics/AuthorAnalyticsPage.tsx

```diff
@@ -280,14 +287,16 @@ const AuthorAnalyticsPage = ({ classes }: { classes: ClassesType }) => {
     <>
       <HeadTags title={title} />
       <SingleColumnSection className={classes.root}>
+      <div className={classes.pageHeader}>
+        <Typography variant="headline" className={classes.pageHeaderText}>
+          Your post stats
+        </Typography>
+      </div>
         <div className={classes.section}>
-          <AnalyticsGraph userId={user._id} title="Stats on all posts" />
+          <AnalyticsGraph userId={user._id}/>
         </div>
         <div className={classes.section}>
           <div className={classes.postsListHeader}>
-            <Typography variant="headline" className={classes.postsListHeaderText}>
-              Posts
-            </Typography>
```

Comment:
A downside of this is that the `checking latest data...` message will now cause a layout shift, but I think it's still worth merging as we may be able to get rid of that in the near future. I'll make a TODO for myself